spark/v3.3/spark/src/jmh/java/org/apache/iceberg/spark/action/IcebergSortCompactionBenchmark.java

```diff
@@ -35,7 +35,7 @@
 import org.apache.iceberg.SortDirection;
 import org.apache.iceberg.SortOrder;
 import org.apache.iceberg.Table;
-import org.apache.iceberg.actions.BinPackStrategy;
```

Comment:
Looks like https://github.com/apache/iceberg/pull/7987 forgot to handle it for Spark 3.3. 

It has to be replaced with proper class now as the deprecated class is removed.